spec/requests/companies_spec.rb

```diff
@@ -4,9 +4,19 @@
 
 RSpec.describe 'Companies', type: :request do
   describe 'GET /index' do
-    it 'returns http success' do
+    it 'render template' do
       get '/companies'
+
       expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('Companies')
+      expect(response.body).to include('Joint Stock Company')
+    end
```

Comment:
I would check that the actual companies are rendered.

```suggestion
    it 'renders the list of companies' do
      Company.create!([
        {
          name: 'MERCATOR',
          symbol: 'PLMRCTR00015',
        },
        {
          name: 'MOSTALPLC',
          symbol: 'PLMSTPL00018',
        },
      ])
      
      get '/companies'

      expect(response).to have_http_status(:success)
      expect(response.body).to include('Companies')
      expect(response.body).to include('MERCATOR')
      expect(response.body).to include('MOSTALPLC')
    end
```

Reply:
Yes, much better :)